pyproject.toml

```diff
@@ -152,6 +152,11 @@ credentials-azure-kv = [
   "azure-keyvault",
   "msrestazure",
 ]
+credentials-azure-mi = [
```

Comment:
is a separate extra still needed?

Reply:
no, I missed this and one other change from the azure_mi work. removing both now